### ISSUE TITLE
Xiaofang S1 Firmware 5.6.2.197 Downgrade

### ISSUE THREAD

**Issue by ZeroNavigator** [EliasKotlyar/Xiaomi-Dafang-Hacks#1067]
Hello everyone.

I have a Xiaofang S1 with T20 chipset at 64mb
Unfortunately i upgrade it to 5.6.2.197 as i did not know the disadvantages yet.

I am kind of new to this but i managed to solder the RxD TxD and GND on a rs232 to usb adapter and with some help from this forum i managed to make a communication using PuTTY on Windows 10.

Tried several times to stop u-boot with no luck.

I will need help on how and if  still possible to downgrade that cams firmware on its last OFW so i can region unlock it and use it normally like before the update.

**Comment by ZeroNavigator** [comment]
So. I tried it using virtual box on a linux environment with the help of putty.
I managed to login with root / ismart12. (maybe there was a keybord issue with windows i dont know)
The thing is.. i did not found a guide with details on what command to use so i can flash an older firmware to it as i am not that familiar with this process..
Does anyone here know a guide/tutorial for this or even explain that should i do next?
